services/navigation/builtin/builtin.go

```diff
@@ -357,7 +370,8 @@ func (svc *builtIn) Close(ctx context.Context) error {
 	return svc.store.Close(ctx)
 }
 
-func (svc *builtIn) startWaypoint(ctx context.Context, extra map[string]interface{}) {
+func (svc *builtIn) startWaypointMode(ctx context.Context, extra map[string]interface{}) {
```

Comment:
If we're changing the name of this function, can we make it something that reflects the permanent nature of its background-ness? `waypointModeBackgroundLoop` or waypointExecutionBackgroundLoop` or something like that, without "start" in the name?

Reply:
its not permanent though... It only lives as long as the nav service is in waypoint mode. I'll create a `stopWaypointMode` to make it clearer that this is indeed not permanent.

Reply:
done